package/wpe/wpeframework-plugins/Config.in

```diff
@@ -965,6 +965,12 @@ config BR2_PACKAGE_WPEFRAMEWORK_TRACECONTROL
     bool "TraceControl"
     help
       TraceControl Plugin
+      
+config BR2_PACKAGE_WPEFRAMEWORK_WARNINGREPORTINGCONTROL
+    select BR2_PACKAGE_WPEFRAMEWORK_RDKSERVICES
+    bool "WarningReportingControl"
```

Comment:
I think we also want to select BR2_PACKAGE_WPEFRAMEWORK_WARNING_REPORTING since it's default switched off